### PR TITLE
Fix #1301 - emission node needs pbr not default

### LINKED ISSUE #1301: Export: emission materials export too bright
A basic emission material like this

emis

exports as

```
  "materials": [
    {
      "emissiveFactor": [1, 0, 0],
      "pbrMetallicRoughness": {},
      ...
    }
```

Since pbrMetallicRoughness is omitted, the defaults are baseColor=[1,1,1,1], metallic=1, roughness=1. This is much too bright.

out

Messing around in Blender to find an equivalent-looking Principled node, baseColor=[0,0,0,1], metallic=(anything), roughness=1 is much better.

> _Note:_ This material setup was discussed at https://github.com/KhronosGroup/glTF-Blender-IO/pull/1209#issuecomment-693649108.

### PR DESCRIPTION
**Pull request by julienduroure**
Fix #1301 
When emission node is used, pbr needs non default values

### CODE REVIEW

File: addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py [hunk 1]
```diff
@@ -60,6 +60,18 @@ def gather_material(blender_material, export_settings):
         pbr_metallic_roughness=__gather_pbr_metallic_roughness(blender_material, orm_texture, export_settings)
     )
 
+    print(material.pbr_metallic_roughness)
+    print("->", orm_texture)
```
Comment (emackey) [inline addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py:64]: Did you intend to leave these `print` statements in here?  There's another one on line 70.

File: addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py [hunk 2]
```diff
@@ -60,6 +60,18 @@ def gather_material(blender_material, export_settings):
...
+
+    # If emissive is set, but no orm_texture,
+    # "Emission" node is used. We need to set manually default values for 
+    # pbr_metallic_roughness.baseColor
+    if material.emissive_factor is not None and orm_texture is None:
```
Comment (emackey) [inline addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py:69]: Is testing for orm_texture the correct answer here?  What if they have a Principled BSDF node without textures?

File: addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py (hunk 1 above)
Comment (julienduroure) [reply addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py]: Fixed

File: addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py (hunk 2 above)
Comment (julienduroure) [reply addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py]: Exact. I just pushed another (better) check

**Review by emackey — APPROVED** [review]
Looks good to me.

**Pull request merged**